details/operator-sdk/controllers/constants.go

```diff
@@ -0,0 +1,129 @@
+package controllers
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// TODO: need to manually update this list if changes are made
```

Comment:
Is there another way of doing this so we don't have to change the cr list in two places. if there was any change in neptune we would then have to change it here too.

Reply:
Yes, technically when we change neptune, we would need to update this list. But adding CRD or bumping versions are few and far between. The purpose for maually defining the list, is so we have control over the ordering of removing CRs as to not to get stuck in a case where the finalizer prevents removal.